common-test-lib/src/main/java/com/google/cloud/tools/intellij/testing/TestUtils.java

```diff
@@ -172,4 +170,54 @@ public static void assertIsSerializable(@NotNull Serializable object) {
       }
     }
   }
+
+
+  /**
+   * Asserts that the given {@link ThrowingRunnable} throws an exception of type {@code
+   * expectedThrowable} when executed.
+   *
+   * @param expectedThrowable the class of the exception that should be thrown
+   * @param runnable the {@link ThrowingRunnable} that should throw
+   * @param <T> the type of exception that should be thrown
+   * @return the thrown exception of type {@code expectedThrowable}
+   * @throws AssertionError if the given {@link ThrowingRunnable} throws a different type of
+   *     exception or does not throw at all
+   */
+  public static <T extends Throwable> T expectThrows(
+      Class<T> expectedThrowable, ThrowingRunnable runnable) {
+    try {
+      runnable.run();
+    } catch (Throwable thrown) {
+      if (isInstanceOfType(thrown, expectedThrowable)) {
+        @SuppressWarnings("unchecked")
+        T retVal = (T) thrown;
+        return retVal;
+      }
+      String message =
+          String.format(
+              "Unexpected exception type thrown; expected '%s' but was '%s'.",
+              expectedThrowable.getSimpleName(), thrown.getClass().getSimpleName());
+      throw new AssertionError(message, thrown);
+    }
+
+    String message =
+        String.format(
+            "Expected '%s' to be thrown, but nothing was thrown.",
+            expectedThrowable.getSimpleName());
+    throw new AssertionError(message);
+  }
+
+  /**
+   * Returns true if the given {@code instance} can be assigned to the given {@code clazz},
+   * otherwise returns false.
```

Comment:
Yep, good catch. I must have gotten lazy. Removed this method though so it's now gone.